### PR TITLE
change remark for FlushAsync method.

### PR DESCRIPTION
**Pull request by stanislavromanov**

### CODE REVIEW

File: src/Microsoft.AspNet.Mvc.Razor/RazorPage.cs [hunk 1]
```diff
@@ -546,8 +546,8 @@ public Task<HtmlString> RenderSectionAsync([NotNull] string name)
         /// </summary>
         /// <returns>A<see cref="Task{HtmlString}"/> that represents the asynchronous flush operation and on
         /// completion returns a <see cref="HtmlString.Empty"/>.</returns>
-        /// <remarks>The value returned is a token value that allows FlushAsync to succeed. However the
-        /// value does not represent the rendered content.</remarks>
+        /// <remarks>The value returned is a token value that allows FlushAsync to  to work directly in the
+        /// an HTML section. However the value does not represent the rendered content.</remarks>
```
Comment (kirthik) [inline src/Microsoft.AspNet.Mvc.Razor/RazorPage.cs]: Could you please fix the comment to say - "The value returned is a token value that allows FlushAsync to work directly in an HTML section."

Currently you have 2 "to"s in the sentence and "the" and "an" before HTML section.

Also, have you signed a [CLA](https://github.com/aspnet/Home/blob/master/CONTRIBUTING.md#contributing-code-and-content) as yet? cc @Eilon

**Comment by kirthik** [comment]
Change looks good. 
Once you sign the [CLA](https://github.com/aspnet/Home/blob/master/CONTRIBUTING.md#contributing-code-and-content), you are good to go, I think. cc @Eilon

**Comment by yishaigalatzer** [comment]
#1721

**Comment by stanislavromanov** [comment]
@kirthik Okey, I did sign the CLA. I hope it's good now and you can accept my pull requests in the future.

**Comment by Eilon** [comment]
CLA is confirmed.

**Comment by harshgMSFT** [comment]
Thanks for fixing this one @stanislavromanov merged : https://github.com/aspnet/Mvc/commit/990473aebc17505a08f59cd85142b30642bc2a54

**Pull request closed without merging**